src/pages/Home/styles/profile-card.css

```diff
@@ -62,14 +62,15 @@
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    font-weight: 450;
```

Comment:
there is a need to fix colors for dark mode, not sure how we should do it but we shall see 👍

Reply:
let's see if we can define some variables so we don't have to call them concurrently. This should be priority before we move on to skills. Once we finalize Dark mode changes, all else should be easy 👯‍♀️